e2e/login.po.ts

```diff
@@ -0,0 +1,67 @@
+import { browser, element, by, ExpectedConditions } from 'protractor';
+import { credentials } from '../test.credentials';
```

Comment:
I assume the format of `test.credentials.js` will be in the form of 

```
export const credentials = {
    username: "username", 
    password: "password" 
};
```

Will it be good if we include an example file to let testers know what information they need to put into their config file? Maybe for example, a `test.crentials.ts.example` at the root directory! :O

Reply:
Yes, that's the format. We can document the format in our dev notes, if we end up adopting this setup.